src/gamma.rs

```diff
@@ -195,6 +198,39 @@ macro_rules! implement { ($kind:ty) => { impl Gamma for $kind {
         let value = unsafe { math::lgamma(self as f64, &mut sign) as Self };
         (value, sign)
     }
+
+    fn trigamma(&self) -> Self {
+
```

Comment:
This empty line is probably not needed. I’d also remove a few lines below so as to have large logical chunks. Not sure. You probably know best. Let’s just make sure that the formatting is consistent across the package.

Reply:
done